options/m_option.c

```diff
@@ -2009,10 +2013,12 @@ static int parse_size_box(struct mp_log *log, const m_option_t *opt,
     return 1;
 
 error:
```

Comment:
I'm not sure if you encountered cases like this in your other fixes like this, but it might be best to rename this label to something like `exit`.